### PR TITLE
Adding cephfs-journal-tool event mode

### PR DESCRIPTION
**Pull request by julpark-rh**
# Description
cephfs-journal-tool event mode

http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-7SESZB/

Please include Automation development guidelines. Source of Test case - New Feature/Regression Test/Close loop of customer BZs
<details>

<summary>click to expand checklist</summary>

- [ ] Create a test case in Polarion reviewed and approved.
- [ ] Create a design/automation approach doc. Optional for tests with similar tests already automated.
- [ ] Review the automation design
- [ ] Implement the test script and perform test runs
- [ ] Submit PR for code review and approve
- [ ] Update Polarion Test with Automation script details and update automation fields
- [ ] If automation is part of Close loop, update BZ flag qe-test_coverage “+” and link Polarion test
</details>

### CODE REVIEW

**Comment by neha-gangadhar** [comment]
@julpark-rh Please work on polarion testcase and move to proposed state.

File: tests/cephfs/cephfs_journal_tool/cephfs_journal_tool_event_mode.py [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+
+from tests.cephfs.cephfs_utilsV1 import FsUtils
+from utility.log import Log
+
+log = Log(__name__)
+
+"""
+Testing description:
+
+Basic funcational test for cephfs-journal-tool event command
```
Comment (sumabai) [inline tests/cephfs/cephfs_journal_tool/cephfs_journal_tool_event_mode.py:13]: functional spell error

File: tests/cephfs/cephfs_journal_tool/cephfs_journal_tool_event_mode.py [hunk 2]
```diff
@@ -0,0 +1,197 @@
...
+
+Steps to Reproduce:
```
Comment (sumabai) [inline tests/cephfs/cephfs_journal_tool/cephfs_journal_tool_event_mode.py:15]: Reproduce word may not be necessary, just 'Test Steps' would suffice

File: tests/cephfs/cephfs_journal_tool/cephfs_journal_tool_event_mode.py [hunk 3]
```diff
@@ -0,0 +1,197 @@
...
+1. mount cephfs using ceph-fuse and kernel
+2. store the journal address and inodes in a list
+3. run "cephfs-journal-tool --rank cephfs:0 event get" with options
```
Comment (sumabai) [inline tests/cephfs/cephfs_journal_tool/cephfs_journal_tool_event_mode.py:18]: Please add on what to validate in each step, after running prescribed command

File: tests/cephfs/cephfs_journal_tool/cephfs_journal_tool_event_mode.py [hunk 4]
```diff
@@ -0,0 +1,197 @@
...
+6. run "cephfs-journal-tool --rank cephfs:0 event get --type" with options
+7. run "cephfs-journal-tool --rank cephfs:0 event splice" with options
+8. Apply function as a problem for now
+"""
+
+
+def run(ceph_cluster, **kw):
+    try:
+        tc = "CEPH-83595482"
+        log.info(f"Running CephFS tests for ceph tracker - {tc}")
```
Comment (sumabai) [inline tests/cephfs/cephfs_journal_tool/cephfs_journal_tool_event_mode.py:30]: ceph tracker ?can it be ceph testcase instead?

File: tests/cephfs/cephfs_journal_tool/cephfs_journal_tool_event_mode.py [hunk 5]
```diff
@@ -0,0 +1,197 @@
...
+            if len(parts) > 4:
+                log.info(parts)
+                journal_address = parts[1]
+                inodes = parts[4]
+                result.append((journal_address, inodes))
+            elif len(line) > 1 and len(line) < 5:
+                result.append((parts[1], ""))
+            else:
+                continue
+        # log.info(f"journal status: {result}")
```
Comment (sumabai) [inline tests/cephfs/cephfs_journal_tool/cephfs_journal_tool_event_mode.py:79]: Can we validate cmd output?

File: suites/quincy/cephfs/tier-4_cephfs_recovery.yaml [hunk 6]
```diff
@@ -218,12 +218,18 @@ tests:
   - test:
       abort-on-fail: false
       desc: "journal_tool_journal_mode"
```
Comment (hkadam134) [inline suites/quincy/cephfs/tier-4_cephfs_recovery.yaml:220]: please add a meaningful description and name about the test.

File: suites/quincy/cephfs/tier-4_cephfs_recovery.yaml [hunk 7]
```diff
@@ -218,12 +218,18 @@ tests:
...
-      module: journal_tool_journal_mode.py
+      module: cephfs_journal_tool.journal_tool_journal_mode.py
       name: journal_tool_journal_mode
       polarion-id: "CEPH-83594249"
   - test:
       abort-on-fail: false
       desc: "journal_tool_header_mode"
```
Comment (hkadam134) [inline suites/quincy/cephfs/tier-4_cephfs_recovery.yaml:226]: please add a meaningful description and name about the test.

File: suites/quincy/cephfs/tier-4_cephfs_recovery.yaml [hunk 8]
```diff
@@ -218,12 +218,18 @@ tests:
...
-      module: journal_tool_header_mode.py
+      module: cephfs_journal_tool.journal_tool_header_mode.py
       name: journal_tool_header_mode
       polarion-id: "CEPH-83594266"
+  - test:
+      abort-on-fail: false
+      desc: "journal_tool_event_mode"
```
Comment (hkadam134) [inline suites/quincy/cephfs/tier-4_cephfs_recovery.yaml:232]: please add a meaningful description and name about the test.

**Review by sumabai — DISMISSED** [review]

**Review by sumabai — APPROVED** [review]

**Review by hkadam134 — APPROVED** [review]
/lgtm

**Review by neha-gangadhar — APPROVED** [review]
Approved

**Pull request merged**